performance-tests/tests/data-files.js

```diff
@@ -0,0 +1,147 @@
+import http from 'k6/http';
+import encoding from 'k6/encoding';
+import { browser } from 'k6/browser';
+import { sleep } from 'k6';
+import { check } from 'https://jslib.k6.io/k6-utils/1.5.0/index.js';
+import { attachAuthCookiesToBrowser, login } from '../utils.js'
+import { Section2File, readAll } from '../files.js'
+
+const file = open('../data/ADS.E2J.FTP2.TS06')
+
+const loadProfiles = {
+  unit: {
+    vus: 1,
+    iterations: 1,
+    executor: 'per-vu-iterations'
+  },
+
+  ramping: {
+    executor: 'ramping-vus',
+    startVUs: 0,
+    stages: [
+      { duration: '20s', target: 2 }, // 0-10vu over 20s
+      { duration: '20s', target: 4 },
+      { duration: '20s', target: 6 },
+      { duration: '20s', target: 8 },
+      { duration: '20s', target: 10 },
+      { duration: '2m', target: 10 },
+      { duration: '40s', target: 0 }
+    ],
+    gracefulRampDown: '60s',
+  }
+}
+
+const scenarios = {
+  browser: {
+    options: {
+      browser: {
+        type: 'chromium',
+      },
+    },
+    exec: 'dataFileBrowserSubmission',
+    ...loadProfiles.unit,
+  },
+  api: {
+    exec: 'dataFileApiSubmission',
+    ...loadProfiles.ramping,
+  }
+}
+
+const SCENARIO = __ENV.SCENARIO || null
+
+export const options = {
+  scenarios: SCENARIO ? { [SCENARIO]: scenarios[SCENARIO] } : scenarios,
+
+  thresholds: {
+    checks: ['rate==1.0'],
+  },
+};
+
+export const dataFileApiSubmission = async () => {
+  login()
+  const httpCookies = http.cookieJar().cookiesForURL(__ENV.BASE_URL)
+
+  const body = {
+    file: http.file(file, 'loadtest-file.txt'),
+    original_filename: 'ADS.E2J.FTP2.TS06',
+    user: 'f94b3a2e-4ee5-4d4b-bab3-20147cb6f480',
```

Comment:
The api tests are failing because the user with this ID does not exist. Could we update the generate cypress user to force this ID? Or, can we make another call to get the ID?
![Screenshot 2024-12-20 at 11 09 25 AM](https://github.com/user-attachments/assets/8935e4a6-b700-41b1-93a7-e7a3ea8dcf06)

Reply:
good catch! i updated it so it pulls the user id from the `login/cypress` endpoint. caveat is that it would only work when logged in as the admin user (unless we update the endpoint)